extensions/wikia/AdEngine/js/spec/lookup/prebid/prebidHelper.spec.js

```diff
@@ -0,0 +1,104 @@
+/*global beforeEach, describe, it, modules, expect, spyOn*/
+describe('ext.wikia.adEngine.lookup.prebid.prebidHelper', function () {
+	'use strict';
+
+	var mocks = {
+			adapters: [
+				{
+					prepareAdUnit: function () {
+						return {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Add a semicolon at the end of this statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3ASemicolon)